.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           - ruby-version: '2.7'
             puppet_gem_version: '~> 7.0'
           - ruby_version: '3.2'
-            puppet_gem_version: 'https://github.com/puppetlabs/puppet' # puppet8'
```

Comment:
Is there a reason for not testing against the branch anymore?

Reply:
hey @bastelfreak this was set pre puppet 8 release to allow us to test our tools on puppet 8. We try to test against the latest stable releases, as opposed to main branch as we have seen merged changes cause problems across our CI before.